spec/purgatory_spec.rb

```diff
@@ -237,6 +237,23 @@
       end
     end
 
+    context "approving a performable method" do
+      before do
+        create_method_call_purgatory
+        @purgatory.soul.stub(:public_send).and_return(false)
```

Comment:
I still think we should change this as per my previous comment:

> I think it would make more sense to stub the return value of the actual method. So I guess `@widget.stubs(:rename).returns(false)`